holoviews/plotting/bokeh/__init__.py

```diff
@@ -197,6 +199,15 @@ def colormap_generator(palette):
                         edge_line_color='black', edge_line_width=2,
                         edge_nonselection_line_color='black',
                         edge_hover_line_color='limegreen')
+options.TriMesh = Options('style', node_size=5, node_line_color='black',
+                          node_fill_color='white', edge_line_color='black',
+                          node_hover_fill_color='limegreen',
+                          edge_hover_line_color='limegreen',
+                          edge_nonselection_alpha=0.2,
+                          edge_nonselection_line_color='black',
+                          node_nonselection_alpha=0.2,
+                          edge_line_width=1)
```

Comment:
That is a lot of options!

Reply:
Agreed, but they're mostly for interactive features when you're selecting or hovering over the graph.